### ISSUE TITLE
Very Small Issue - icon.png Missing

### ISSUE THREAD

**Issue by Superkat32** [n643064/Apocalypse#2]
**TLDR:** Mod icon is broken, and I think it would be nice if that could be fixed next update.

Hello! I've been using this mod for a week or two now, and I've really enjoyed how it changes the game. The only issue I've found with it is that the icon is missing in-game. This is very small, but I use the icons of mods to distinguish them from each other whenever attempting to change the config, so I think it would be nice if the icon wasn't broken.

This should be a very easy fix. You already have an icon designed because there is one on the Modrinth project, so all it should take is copying over the icon into the right spot, then it should work(assuming it's the correct size and format). A fix for something like this is probably too small to be an update within itself, so you could just throw it in the next update whenever that comes around.

(Screenshot for reference, but you're probably already aware of this whenever testing the mod)

![image](https://user-images.githubusercontent.com/89557012/222319472-c5ac1103-b047-4b91-acb0-a7894ccf6023.png)

**Comment by n643064** [comment]
I'm aware of this; will add the icon in the next release.

**Comment by Superkat32** [comment]
Alrighty, thanks!
